volim.html

```diff
@@ -100,5 +100,21 @@
 			<p class="openhour-lesson__dance">Tab Dancing : 19:00 - 23.00</p>
 		</div>
 	</div>
+  <div class="shortcode">
+   <div class="shortcode-top">
+      <div class="shortcode-title">
+        <p class="shortcode-title__text">SHORTCODE</p>
```

Comment:
please add `uppercase` modification for text in css, in case data would be in lower case